scripts/plot_variables.py

```diff
@@ -100,7 +100,7 @@ def plot_lc_table(flux_table, stats_table, start=0, stride=1, plot_dir="plots",
         flux_mask = list(np.choose(mask, fluxes))
         err_flux_mask = list(np.choose(mask, err_fluxes))
         if dates:
-            epoch_times = [datetime.datetime.strptime(a, "%Y-%m-%dT%H:%M:%S") for a in list(row[epochs][epoch_mask])]
+            epoch_times = [datetime.datetime.strptime(a.split(".00")[0], "%Y-%m-%dT%H:%M:%S") for a in list(row[epochs][epoch_mask])]
```

Comment:
Stripped out trailing zeroes in the datetime. For example, this:

`2013-09-10T20:47:04.00000` 

to:

`2013-09-10T20:47:04`

otherwise there will be an error in `datetime.datetime.strptime` that there is unconverted portions.

Reply:
I can avoid this by putting a fixed string length in the Table column creation above that's the exact Epoch length, in pseudocode:

` new_cols.append(Column(data=str_data.copy(), name=colname.format(i), dtype=len(epoch_string))`

Unfortunately I think it'll have to be manual as the table is constructed before we know the length.